LogIn/UIExtensions.swift

```diff
@@ -5,23 +5,31 @@
 //  Created by Aleksei Permiakov on 07.12.2021.
 //
 
+
 import UIKit
 
 extension UITextField {
     func loginTextStyle() {
```

Comment:
maybe something like `applyLoginTextStyle`, because meaning is not clear

Reply:
done